app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsViewModel.kt

```diff
@@ -18,6 +18,7 @@ class DeveloperOptionsViewModel @Inject constructor(
   private val activity: AppCompatActivity,
```

Comment:
Nit update
`activity: AppCompatActivity,`

Reply:
Same as the above comment.